### DISCUSSION TITLE
[Adonis 5] How to query database on app start and keep the values?

### DISCUSSION THREAD

**Discussion by guntribam** [adonisjs/core#3223] [Help] [answered]
I need to query my db on a table that contains user actions in order to dynamically create Bouncer actions.

How can i query on app start in order to provide Bouncer an array of profiles in every validation?

**Comment by arthur-er** [comment, marked as answer]
Use a preload file.

```bash
$ node ace make:prldfile loadActions
```

```typescript
// start/loadActions.ts
// query the db here, and update the needed values.
```

**Reply by guntribam** [reply]
thanks
